pr-data.csv

```diff
@@ -3047,7 +3047,8 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testUserIdentityMapping,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-ldap-iaa-providers-bundle/nifi-ldap-iaa-providers,org.apache.nifi.ldap.tenants.LdapUserGroupProviderTest.testUserIdentityMappingWithTransforms,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,minifi/minifi-toolkit/minifi-toolkit-configuration,org.apache.nifi.minifi.toolkit.configuration.ConfigMainTest.testTransformRoundTripMultipleRelationships,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,minifi/minifi-toolkit/minifi-toolkit-configuration,org.apache.nifi.minifi.toolkit.configuration.ConfigMainTest.testTransformRoundTripNestedControllerServices,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,minifi/minifi-toolkit/minifi-toolkit-configuration,org.apache.nifi.minifi.toolkit.configuration.ConfigMainTest.testTransformRoundTripNestedControllerServices,ID,Deleted,,https://github.com/apache/nifi/commit/3aa02a022e154416cc6f69acc327aeb26f2a89b2
+
```

Comment:
There should be no empty line. (You could open a PR on the checker to give a better error message.)